src/Validator.php

```diff
@@ -719,7 +719,7 @@ private static function validateUrlComponents(array $components): array
     }
 
     /**
-     * Helper method for the url and absoluteUrl methods.
+     * Helper method for the URL and absoluteUrl methods.
```

Comment:
Refers to a lowercase method name.

Reply:
Well, I tend to append ``()`` behind method names and ``$`` before property names to better distinguish them from each other and other syntax.